index.bs

```diff
@@ -729,23 +729,23 @@ The {{Element/requestFullscreen|Element.requestFullscreen()}} method steps are u
 ## Permission API Integration ## {#api-permission-api-integration}
 <!-- ====================================================================== -->
 
-This specification defines a [=/default powerful feature=] that is identified by the [=powerful feature/name=] "<dfn export permission>window-placement</dfn>".
+This specification defines a [=/default powerful feature=] that is identified by the [=powerful feature/name=] "<dfn export permission>window-management</dfn>".
 
 The [[permissions]] API provides a uniform way for websites to query the state of their permissions.
 
-Issue: Add {{window-placement}} to [[permissions]] registry.
+Issue: Add {{window-management}} to [[permissions]] registry.
 
 Issue: Define behavior of cached objects and method steps when the permission is revoked. (See [#80](https://github.com/w3c/window-placement/issues/80))
 
 <!-- ====================================================================== -->
 ## Permission Policy integration ## {#api-permission-policy-integration}
```

Comment:
Ditto: This section should document the rename, and plans to phase out support for the legacy string.